spec/rails_helper.rb

```diff
@@ -8,6 +8,9 @@
 if RUN_SYSTEM_SPECS
   STREAMING_PORT = ENV.fetch('TEST_STREAMING_PORT', '4020')
   ENV['STREAMING_API_BASE_URL'] = "http://localhost:#{STREAMING_PORT}"
+else
+  # Fix specs writing to public/system
+  ENV['PAPERCLIP_ROOT_PATH'] ||= File.join('spec', 'test_files')
```

Comment:
Upstream removed `RUN_SYSTEM_SPECS`, so this will break.
Not sure how to handle that yet.